src/OrchardCore.Modules/OrchardCore.ContentFields/Views/TextField-Monaco.Edit.cshtml

```diff
@@ -18,7 +18,7 @@
     </div>
 </div>
 
-<script asp-name="monaco" depends-on="admin" at="Foot"></script>
+<script asp-name="monaco" depends-on="jQuery" at="Foot"></script>
```

Comment:
I don't think mocaco needs to depend on `jQuery`. Why do you think this is needed? I think all you have to do here is change line 23 from `$(document).ready(function () {` to `document.addEventListener('DOMContentLoaded', () => {`
Can you please give it a try? I think we should try to remove as much of the jQuery dependency as possible. Yes maybe in a separate PR. But, since you are adding a direct dependency to jQuery, maybe you should re-evaluate it.